Cargo.toml

```diff
@@ -129,7 +129,7 @@ toml = { version = "0.8", optional = true, default-features = false, features =
 serde_json = { version = "1.0", optional = true }
 yaml-rust2 = { version = "0.10", optional = true }
 rust-ini = { version = "0.21", optional = true }
-ron = { version = "0.8", optional = true }
+ron = { version = "0.9", optional = true }
```

Comment:
> 128 integer support in Ron [requires feature opt-in via Cargo.toml](https://github.com/ron-rs/ron/pull/479/files#diff-5648b11aa0ef7c281cdd79add2b554fdb305a5ffdb4a205a8923f0ded22672f7), (as [ValueKind supports an equivalent type](https://github.com/mehcode/config-rs/blob/55c464e143d7301f3c107a405827fe9dc085f81a/src/value.rs#L20-L22), config-rs could opt-in).

This actually has a significant problem and we might want to wait to see how ron-rs/ron#563 gets resolved before upgrading.

Reply:
That issue seems to have been resolved. Iiuc, updating to `0.10` fixes the problem?

Reply:
0.10 does not fix the problem